treeherder/config/settings.py

```diff
@@ -426,6 +418,14 @@
 # Halves the time spent performing Brotli/gzip compression during deploys.
 WHITENOISE_KEEP_ONLY_HASHED_FILES = True
 
+# Templating
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'APP_DIRS': True,
+        'DIRS': [WHITENOISE_ROOT],
```

Comment:
For when the server is looking for the index file to return for any of our routes.